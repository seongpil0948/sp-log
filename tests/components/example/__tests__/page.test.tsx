import '@testing-library/jest-dom'
import React from 'react'

import { render, screen } from '@testing-library/react'

import Page from '../TestPage'
 
describe('Page', () => {
  it('renders a heading', () => {
    render(<Page />)
 
    const heading = screen.getByRole('heading', { level: 1 })
 
    expect(heading).toBeInTheDocument()
  })
  
  it('renders homepage unchanged', () => {
    const { container } = render(<Page />)
    expect(container).toMatchSnapshot()
  })  
})