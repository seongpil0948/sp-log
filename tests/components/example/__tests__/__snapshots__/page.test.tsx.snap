// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      Home
    </h1>
    <a
      href="/about"
    >
      About
    </a>
  </div>
</div>
`;
